packages/block-library/src/columns/editor.scss

```diff
@@ -9,13 +9,6 @@
 	}
 }
 
-// Ideally all block toolbars should be positioned the same.
-.components-popover.block-editor-block-list__block-popover
-.components-popover__content
-.block-editor-block-contextual-toolbar[data-type="core/column"] {
-	margin-left: 0;
-}
-
```

Comment:
What was the purpose of these styles?

Reply:
I don't know @jasmussen  maybe, It was definitely breaking the column block toolbar position though.

Reply:
These styles were almost certainly necessary due to the complex left/right margins that innerblock containers used to be born with. 

But it's easy to test, if the toolbar is positioned correctly according to the block selected, then these styles are not needed anymore. And good riddance!